sphinxcontrib/video.py

```diff
@@ -184,11 +188,16 @@ def run(self):
 
 def visit_video_node_html(translator: HTMLTranslator, node: video_node) -> None:
     """Entry point of the html video node."""
+    # align
+    if node["align"] and (node["align"] in ["left", "center", "right"]):
+        html: str = f"<div class=\"align-{node['align']}\">"
+    else:
+        html: str = f"<div class=\"align-default\">"
```

Comment:
Considering that "default" is set as default value for the align option, This could be transformed into: 

 ```suggestion
html = f"<div class=\"align-{node['align']}\">"
```

The open questio being should we raise a warning if the align parameter is not in ["left", "right", "center", "default"] ? I think yes